src/test/java/uk/gov/hmcts/reform/sscs/personalisation/PersonalisationTest.java

```diff
@@ -1587,6 +1587,24 @@ public void setHelplineTelephoneNumber_relevantToTheCaseCountry(String isScottis
         assertEquals(helpLineTelephone, result.get(HELPLINE_PHONE_NUMBER));
     }
 
+    @Test
+    @Parameters({"PIP", "DLA"})
+    public void testPanelCompositionByBenefit(String benefitType) {
```

Comment:
There is already a test called: customisePersonalisation() around line 488, would it not be better to extend that as some of this looks duplicated?